docs/services/aws-services-guide.md

```diff
@@ -76,7 +83,7 @@ The output of this command will look like:
 >}
 >```
 
-Creating the stack continutes in the background and  could take a few minutes to complete. You can check its status by running:
+Creating the stack continues in the background and  could take a few minutes to complete. You can check its status by running:
```

Comment:
Nitpick: `and  could` has two spaces; it should be `and could` with a single space